README.md

```diff
@@ -48,6 +48,7 @@ Documentation
 * [Installation](docs/Installation.md)
 * [Configuration](docs/Configuration.md)
 * [Usage](docs/Usage.md)
+* [Environment Variables](docs/Env_variables.md)
```

Comment:
Did you intend to add a markdown file called `Env_variables.md` and forget to add it to the commit, or was this supposed to link to the new section in `Configuration.md`

Reply:
you're right, I changed my mind mid course 😅 
I'll remove that and merge.

Thanks!